src/network_formatting.jl

```diff
@@ -0,0 +1,43 @@
+matrix_to_list_of_list(A::SparseMatrixCSC) = matrix_to_list_of_list(eltype(A.rowval),A)
+
+function matrix_to_list_of_list(::Type{S}, A::SparseMatrixCSC) where S 
```

Comment:
There is a subtle bug here, this is correct for undirected graphs, but not for directed graphs since that will transpose in-edges to out-edges.

Reply:
Not sure what the best solution for this is. When I wrote them it was intended for undirected graphs because the 
`forest_fire_graph` is expecting an undirected input, and generates new edges with the intention of symmetrizing them. 

I can either 
1. Rename `list_of_list_to_matrix`to reference they’re for undirected neighbor lists 
2. Write a `_get_inedges` for MatrixNetworks to be more careful, but we’d also need to rename the `list_of_list_to_matrix` function to better express what the output is, because it won’t be an undirected neighbor list.
3. Convert MatrixNetwork to a SparseCSCMatrix before calling the code, but this seems like a bad solution as the package is about the MatrixNetwork.

Reply:
The standard solution in MatrixNetworks is just to take a transpose... if they want to avoid that, then they use a MatrixNetwork type instead.

Reply:
ah, I was thinking about the wrong function! Ok that works